apps/core/scenes/search/Search.js

```diff
@@ -26,33 +16,17 @@ import {
 } from '@kiwicom/margarita-navigation';
 import * as DateFNS from 'date-fns';
 
-const TRIP_TYPE = {
-  return: {
-    icon: 'flight-return',
-    label: 'Return',
-  },
-  oneWay: {
-    icon: 'flight-direct',
-    label: 'One way',
-  },
-};
-
-const MODAL_TYPE = {
-  HIDDEN: 'HIDDEN',
-  TRIP_TYPE: 'TRIP_TYPE',
-  PASSENGERS: 'PASSENGERS',
-};
-
-const DATEPICKER_MODE = {
-  DEPARTURE: 'DEPARTURE',
-  RETURN: 'RETURN',
-};
+import Placepickers from './placepickers/Placepickers';
```

Comment:
Yes, you are right, when I made these two components, I was imaging it would have more child components. However they do not now. Maybe in the future though. But yes, I will move them out of the subfolder.